guides/common/assembly_configuring-and-setting-up-remote-jobs.adoc

```diff
@@ -1,4 +1,4 @@
-ifdef::context[:parent-context: {context}]
+:parent-context: {context}
```

Comment:
If the modules do not change context themselves, which they shouldn't, then this is not needed at all.